backend/routes/fundingItems.js

```diff
@@ -0,0 +1,37 @@
+const router = require('express').Router();
+let FundingItem = require('../models/FundingItem.model');
+
+router.route('/').get((req, res) => {
```

Comment:
lets use _ instead of req to avoid unused values, as I'll be adding a check for that in the formatter